### ISSUE TITLE
FormBuilder form_widget() is rendering the error message

### ISSUE THREAD

**Issue by lebadapetru** [symfony/symfony#27453]
Symfony version im using 4.1.0

During development of an app i was using form_row() to render each input with its label and errors, but when i tried to customize it a bit(i wanted the error message to be shown in the bottom of the input) i switched  to using form_label(),form_widget(),form_errors(). Once i test it i figured the error message has been duplicated(one above the input and one below most likely rendered by form_errors).
(Yes i did cleared the cache.)

SetupWizardType.php (form class) 

```
public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('language', EntityType::class, [
                'translation_domain' => 'labels',
                'label' => 'label.languages',
                'placeholder' => 'label.choose',
                'class' => Languages::class
            ])
            ->add('country', EntityType::class,[
                'translation_domain' => 'labels',
                'label' => 'label.countries',
                'placeholder' => 'label.choose',
                'class' => Countries::class
            ])
            ->add('timezone', EntityType::class,[
                'translation_domain' => 'labels',
                'label' => 'label.timezones',
                'placeholder' => 'label.choose',
                'class' => Timezones::class
            ])
            ->add('currency', EntityType::class,[
                'translation_domain' => 'labels',
                'label' => 'label.currencies',
                'placeholder' => 'label.choose',
                'class' => Currencies::class
            ]);

    }
```

setup_wizard.twig.html

```
{{ form_start(form, {method: 'POST', action: path('wizard_index')}) }}

 {{ form_label(form.language) }}
 {{ form_widget(form.language, {attr: {'onchange': 'this.form.submit();'}}) }}
{{ form_errors(form.language) }}

{{ form_label(form.country) }}
{{ form_widget(form.country) }}
 {{ form_errors(form.country) }}

{{ form_label(form.timezone) }}
{{ form_widget(form.timezone) }}
{{ form_errors(form.timezone) }}

{{ form_label(form.currency) }}
{{ form_widget(form.currency) }}
 {{ form_errors(form.currency) }}
    <div class="form-group">
            <button class="btn">{% trans from 'buttons' %}button.prev_step{% endtrans %}</button>
            <button class="btn">{% trans from 'buttons' %}button.next_step{% endtrans %}</button>
            <input class="btn" type="submit" name="save" value="{% trans from 'buttons' %}button.save{% endtrans %}" formnovalidate>
                        </div>

                    {{ form_end(form) }}
```

If i remove every form_errors() , the message will show only once above the input, that's why i supposed it is rendered by form_widget.
Any thoughts on this?

**Comment by xabbuh** [comment]
I am sorry, but I have to close your issue as we use GitHub issues only to discuss new features and for bug reports. For support, please refer to one of the [support channels](https://symfony.com/support). Thank you for understanding.

**Comment by lebadapetru** [comment]
well, that's the problem... i asked for help on slack yet we couldn't figure out what was wrong,so i thought it was a bug,that's why i started a thread

**Comment by xabbuh** [comment]
Sorry, I think I then did not properly understand that. Could you create a small example application that allows to reproduce your issue?

**Comment by lebadapetru** [comment]
I just commited everything on my repo(https://github.com/lebadapetru/learningSymfony) ,the form rendering takes place in templates>setup_wizard.html.twig . The data is sent to the view through the SetupWizardController.php where i'm using the form class SetupWizardType.php to build the form.

**Comment by lebadapetru** [comment]
Any news here? Is it a bug or something wrong in my code?

**Comment by xabbuh** [comment]
@lebadapetru I tried to run your project, but gave up after some time because of config errors and missing classes/environment variables. However, I took a look at your config. You are using the Bootstrap 4 form theme where form errors are indeed rendered with the label: https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig#L184-L215 So yes if I do not misunderstand what you are doing, this is indeed the expected behaviour. So I am going to close as there is nothing to fix.
